packages/cli/src/commands/generate/helpers.js

```diff
@@ -141,6 +141,12 @@ export const yargsDefaults = {
   },
 }
 
+export const validateName = (name) => {
+  if (name.match(/^\//)) {
```

Comment:
I tested ` and it won't let you generate a page. I tested . and , and it breaks, so we should probably add in a check for . as well.

```
      <Route path="/" page={,Page} name="," />
      <Route path="/" page={.Page} name="." />

Reply:
Updated the check, these should show an error now!